tests/e2e/br/br.go

```diff
@@ -215,6 +215,60 @@ var _ = ginkgo.Describe("Backup and Restore", func() {
 			})
 		}
 	})
+
+	ginkgo.Context("[Backup Clean]", func() {
+		ginkgo.It("clean bakcup files with policy Delete", func() {
+			backupClusterName := "backup-clean"
+			backupVersion := utilimage.TiDBLatest
+			enableTLS := false
+			dbName := "e2etest"
+			backupName := backupClusterName
+			typ := strings.ToLower(typeBR)
+
+			ns := f.Namespace.Name
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			ginkgo.By("Create TiDB cluster for backup")
+			err := createTidbCluster(f, backupClusterName, backupVersion, enableTLS)
+			framework.ExpectNoError(err)
+
+			ginkgo.By("Wait for backup TiDB cluster ready")
+			err = utiltidbcluster.WaitForTidbClusterConditionReady(f.ExtClient, ns, backupClusterName, tidbReadyTimeout, 0)
+			framework.ExpectNoError(err)
+
+			ginkgo.By("Forward backup TiDB cluster service")
+			backupHost, err := portforward.ForwardOnePort(ctx, f.PortForwarder, ns, getTiDBServiceResourceName(backupClusterName), 4000)
+			framework.ExpectNoError(err)
+			err = initDatabase(backupHost, dbName)
+			framework.ExpectNoError(err)
+
+			ginkgo.By("Write data into backup TiDB cluster")
+			backupDSN := getDefaultDSN(backupHost, dbName)
+			err = blockwriter.NewDefault().Write(context.Background(), backupDSN)
+			framework.ExpectNoError(err)
+
+			ginkgo.By("Create RBAC for backup and restore")
+			err = createRBAC(f)
+			framework.ExpectNoError(err)
+
+			ginkgo.By("Create backup with clean policy Delete")
+			backup, err := createBackupAndWaitForComplete(f, backupName, backupClusterName, typ, func(backup *v1alpha1.Backup) {
```

Comment:
Suggest including the cleanOptions in the case.

Reply:
OK, I'll add that later

Reply:
We have default values for the clean options, so it should be OK not to add them explicitly.